### DISCUSSION TITLE
k3s example warnings on copyToHost

### DISCUSSION THREAD

**Discussion by strobelight** [lima-vm/lima#1451] [Q&A] [answered]
I'm getting `WARN[0000] Non-strict YAML is deprecated and will be unsupported in a future version of Lima` when starting the k3s.yaml example. The warning also contains `error="unknown field \"copyToHost\"\n "`. 

Is there another way to get `/etc/rancher/k3s/k3s.yaml` copied to the host for use with kubectl?

**Comment by afbjorklund** [comment, marked as answer]
You can use `sudo cat`, I believe ?

```shell
mkdir -p ~/.lima/k3s/copied-from-guest
limactl shell k3s sudo cat /etc/rancher/k3s/k3s.yaml > ~/.lima/k3s/copied-from-guest/kubeconfig.yaml
```

**Reply by strobelight** [reply]
true, just checking whether or not the example is broke now (thought it used to work)

**Reply by afbjorklund** [reply]
It might be so that the examples on master only work with the code on master

**Reply by strobelight** [reply]
ok, I see, I need to checkout the tag for 0.14.1 since that's the latest that brew installs. I see `copyToHost` doesn't exist.

**Reply by AkihiroSuda** [reply]
The latest version is v0.15
